stubs/RdKafka.php

```diff
@@ -81,4 +74,37 @@ public function setLogLevel($level)
     public function queryWatermarkOffsets($topic, $partition, &$low, &$high, $timeout_ms)
     {
     }
+
+    /**
+     * @param array $topicPartitions
+     * @param int   $timeout_ms
```

Comment:
This comment is irrelevant because you specified all the types in the method definition and you can remove it.